sys/mips/gt64120.c

```diff
@@ -346,6 +346,8 @@ static resource_t *gt_pci_alloc_resource(device_t *dev, res_type_t type,
     /* The size of allocated resource must not be larger than what was reported
      * by corresponding BAR. */
     size = min(size, bar->size);
+    /* XXX: Perhaps, the rman_alloc_resource should take this into account */
+    size = roundup(size, PAGESIZE);
```

Comment:
Please consult that with @mohrcore, @ThomasStachovsky, @MichalBlk or @cahirwpz.

Reply:
This PR is blocked until merging #826 .